package.json

```diff
@@ -43,6 +43,7 @@
   },
   "dependencies": {
     "@types/node": "^10.14.22",
+    "base64url": "^3.0.1",
```

Comment:
I opted for the package since I also needed to decode base64url (https://github.com/interledgerjs/ilp-protocol-stream/pull/133/commits/757ccff5ce39858abbc0541092caba6fdb18bbad#diff-916156c481141bfcd08c6ae40ce10e00R219) which isn't as concise.
Assuming I'll still need to decode for how we end encoding the receipt secret and key in the destination address (https://github.com/interledgerjs/ilp-protocol-stream/pull/133#discussion_r395326204), I can plan on adding base64url decode functionality instead of using the package.

Reply:
Isn't it just:

```
function decodeBase64 (string) {
  return Buffer.from(string
      .replace(/-/g, '+')
      .replace(/_/g, '/'),
    'base64')
}
```

Reply:
ah, you're right. I was looking at the library's implementation without thinking.
All the more reason not to use it.